tier3/checklist.md

```diff
@@ -0,0 +1,441 @@
+
+| Tool | Description | Link |
+|------|-------------|------|
+|Repo Linter| Lint repositories for common issues such as missing files,etc...| https://github.com/todogroup/repolinter|
+| Gitleaks | Protect and discover secrets using Gitleaks🔑 | https://github.com/gitleaks/gitleaks |
+| git filter-repo    | Entirely remove unwanted files / files with sensitive data from a repository's history | https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/removing-sensitive-data-from-a-repository |
+
+#### Results
+*Insert Review Here*
+
+
+### Review Licensing
+
+Ensure that acceptable licensing is decided for the project. Most often, software released
+as open source by the federal government does so under the Creative Commons Zero 1.0
+license.
+
+Suggested licensing:
+
+ **Public Domain**
+
+ This project is in the public domain within the United States, and copyright and related rights in the work worldwide are waived through the CC0 1.0 Universal public domain dedication.
+
+ All contributions to this project will be released under the CC0 dedication. By submitting a pull request, you are agreeing to comply with this waiver of copyright interest.
+
+#### Results
+*Insert Review Here*
+
+
+### Review Commit History
+
+Review the history of commits to the version control system used, and whether the team
+prefers to clean (e.g., rebase) this history before releasing to the public.
+
+If not rebasing, verify that:
+1. there are no obscene or impolite remarks in comments or commit history
+2. there are no sensitive internal URLs/IP Addresses in comments or commit history
+3. there are no credential files such as Passwords, API/SSH/GPG keys checked into
+the repo.
+
+Consider using the following tools to perform the tasks above:
+
+ Tool | Description | Link |
+|------|-------------|--------------------|
+| gitleaks | Open source tool that detects and prevents secrets (passwords/api/ssh keys) checked-in to your git repo | https://github.com/gitleaks/gitleaks <br> [What is git leaks and how to use it?](https://akashchandwani.medium.com/what-is-gitleaks-and-how-to-use-it-a05f2fb5b034) |
+| git filter-repo | Entirely remove unwanted files / files with sensitive data from a repository's history | https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/removing-sensitive-data-from-a-repository |
+
+#### Results
+*Insert Review Here*
+
+
+### Review Repository Hygiene
+As part of our repository hygiene requirements, the project must include certain files and
+sections. Using repolinter will help you identify missing files and content that will need to
+be added to your repository before outbounding.
+
+#### Running repolinter on your repository locally
+1. Add [repolinter.json](https://github.com/DSACMS/repo-scaffolder/blob/main/tier3/%7B%7Bcookiecutter.project_slug%7D%7D/repolinter.json) to the root directory of your project
+2. Run command: repolinter lint .
```

Comment:
We would like to use a code block here instead of regular text! Follow what Jabir did here for this: https://github.com/DSACMS/repo-scaffolder/blob/dev/tier1/checklist.md?plain=1#L191-L193

Reply:
https://github.com/DSACMS/repo-scaffolder/blob/dev/tier1/checklist.md for how it looks like